basiclibrary/src/main/java/basiclibrary/Library.java

```diff
@@ -0,0 +1,70 @@
+/*
+ * This Java source file was generated by the Gradle 'init' task.
+ */
+package basiclibrary;
+
+import java.lang.reflect.Array;
+import java.util.Random;
+
+public class Library {
+    public boolean someLibraryMethod() {
+        return true;
+    }
+
+    public int [] roll(int n) {
+        int [] resultArray = new int[n];
+        for(int i=0; i<n; i++) {
+            resultArray[i] = (int) (Math.random() * 6 + 1);
+            System.out.println("Rolled Values: "+ resultArray[i]);
+        }
+        return resultArray;
+    }
+
+    public boolean containsDuplicates(int [] numberArray) {
+        boolean duplicateFlag = false;
+        for (int i = 0; i < numberArray.length; i++) {
+            for (int j = i + 1; j < numberArray.length; j++) {
+                if (numberArray[i] == numberArray[j]) {
+                    duplicateFlag = true;
+                    break;
```

Comment:
Instead of using the `break` statement here, you can just `return true` instead.